README.md

```diff
@@ -7,22 +7,301 @@ Serlio requires so-called rule packages (RPK) as input, which are authored in Ci
 Serlio is well suited for managing the procedural generation of architectural 3D content in digital sets. However, Serlio is restricted to the procedural generation of single buildings / objects. Serlio does not include the city layouting and street network editing tools of CityEngine i.e. the rich CityEngine toolset to design a city from scratch (or based on geographic data) is still needed.
 
 
-## Documentation
+## Quick Start
+1. Use CityEngine to [export a Rule Package](https://doc.arcgis.com/en/cityengine/latest/help/help-rule-package.htm) (RPK).
```

Comment:
I would leave this away in favor of attaching an "extrude.rpk" to the release (see below).

Reply:
removed paragraph and added a extrude.rpk to doc/data/extrude.rpk